src/main/groovy/jenkins/automation/utils/CommonUtils.groovy

```diff
@@ -106,6 +106,39 @@ class CommonUtils {
     }
 
     /**
+     * Utility to add a performance publisher block, for use in performance tests
+     *
+     * @see <a href="https://github.com/cfpb/jenkins-automation/blob/gh-pages/docs/examples.md#common-utils" target="_blank">Common utils</a>
+     */
+
+    static void addPerformancePublisher(context, String reportPattern="**/results/*.jtl", String unstableResponseTimeThreshold="", int failedThresholdPositive, int failedThresholdNegative, int unstableThresholdPositive, int unstableThresholdNegative) {
```

Comment:
Some of these are defaulted. I think it is ok to have as a function, not a class.